src/status_im/mailserver/core.cljs

```diff
@@ -504,22 +507,27 @@
   (when-not (zero? (:peers-count db))
     (if-let [preferred-mailserver (preferred-mailserver-id db)]
       (let [current-fleet (node/current-fleet-key db)]
+        ;; Error connecting to the mail server
         {:db
-         (update-mailserver-state db :error)
-         :ui/show-confirmation
-         {:title               (i18n/label :t/mailserver-error-title)
-          :content             (i18n/label :t/mailserver-error-content)
-          :confirm-button-text (i18n/label :t/mailserver-pick-another)
-          :on-accept           #(re-frame/dispatch
-                                 [:navigate-to (if platform/desktop?
-                                                 :advanced-settings
-                                                 :offline-messaging-settings)])
-          :extra-options       [{:text    (i18n/label :t/mailserver-retry)
-                                 :onPress #(re-frame/dispatch
-                                            [:mailserver.ui/connect-confirmed
-                                             current-fleet
-                                             preferred-mailserver])
-                                 :style   "default"}]}})
+         (update-mailserver-state db :error)}
+        (let [error-dismissed (connection-error-dismissed db)]
```

Comment:
Shall we call this `error-dismissed?` instead? It seems to be common practice to suffix boolean values with `?`. Or does this only apply to certain booleans?

Reply:
yes its better to use `?`

Reply:
Thanks for letting me know, I'll change it.